app/assets/templates/single-post-viewer/single-post-content_tpl.jst.hbs

```diff
@@ -39,11 +39,11 @@
             {{/if}}
             <span class="post-time">
               {{#if root}}
-                <a href="/posts/{{root.id}}">
+                <a href="/posts/{{root.guid}}">
```

Comment:
Yes, we should use `guid` on the line 94.

Now I think that for this line (42)  we can leave `root.id` instead of `root.guid`, since it's more likely to be used directly to get the original post, so it may affect performance.